cc.c

```diff
@@ -196,7 +169,7 @@ int main(int argc, char** argv)
 		}
 		else if(match(argv[i], "-h") || match(argv[i], "--help"))
 		{
-			fputs(" -f input file\n -o output file\n --help for this message\n --version for file version\n", stdout);
+			fputs("Usage: M2-Planet [options] file...\nOptions:\n --file,-f                      input file\n --output,-o                    output file\n --architecture,-A ARCHITECTURE Target architecture. Call without argument to list available\n -D                             Add define\n -E                             Preprocess only\n --debug,-g                     Debug mode\n --bootstrap-mode               Emulate less powerful cc_* compilers\n --max-string N                 Size of maximum string value (default 4096)\n --help,-h                      Display this message\n --version,-V                   Display compiler version\n\n", stdout);
```

Comment:
Maybe see if you can split it into multiple lines like I showed in M2libc example (in your M2-Mesoplanet PR?)

Reply:
I just tested with `cc_amd64` and it doesn't seem to have string literal string concatenation working. It seems to work for `M2-Planet` so I'll change it for `M2-Mesoplanet`.

Reply:
I'll change it to be multiple `puts` calls instead. It will make it easier to add new lines and keep indentation correct. The extra bloat likely won't be a problem.

Reply:
Fixed and rebased onto new master.

Reply:
Added error if using string literal concatenation in bootstrap mode. Let me know if I'm using holding `cc_amd64` wrong and it actually supports it.

Reply:
Hmm, strange, this file is definitely used by cc_amd64: https://github.com/oriansj/M2libc/blob/main/amd64/linux/bootstrap.c and it has the same string behaviour.

Reply:
Rebased onto new master.

That's weird. It's not a special syntax inside the `asm` block?

Reply:
Hmm, yeah, that might be true...